utilities/cert_utility.cc

```diff
@@ -23,7 +23,7 @@ DEFINE_bool(print_all, false, "verbose");
 DEFINE_string(operation, "", "generate policy key and self-signed cert");
 
 DEFINE_string(policy_key_name, "policyKey", "key name");
-DEFINE_string(policy_key_type, "rsa-2048-private", "policy key type");
+DEFINE_string(policy_key_type, enc_method_rsa_2048_private, "policy key type");
```

Comment:
The main reason why we prefer (actually, have to) declare these constants as `char *` rather than `const String` is that this initializer seems to not work if these constants are declared as `String`. 

At run-time, strangely, `FLAGS_policy_key_type` will end up as `""`.

I cannot also initialize this here using `enc_method_rsa_2048_private.c_str()` as that raises a compile-time exception in these initialization macros.